nlu_hyperopt/app.py

```diff
@@ -53,7 +69,14 @@ def worker_function(space):
     best_config = space_eval(search_space, best)
     logger.debug("The best values are: {}".format(best_config))
 
-    data_dir = os.environ.get("DATA_DIRECTORY", "/data")
+    data_dir = os.environ.get("INPUT_DATA_DIRECTORY", "./data")
     with open(os.path.join(data_dir, "template_config.yml")) as f:
         config_yml = f.read().format(**best_config)
         logger.info("The best configuration is: \n{}\n".format(config_yml))
+
+        # ## For github action
+        config_yml=config_yml.replace('%','%25') ## github actions does not handle multiline outputs properly
+        config_yml=config_yml.replace('\n','%0A') ## https://github.community/t5/GitHub-Actions/set-output-Truncates-Multiline-Strings/td-p/37870
+        config_yml=config_yml.replace('\r','%0D')
+
+        print(f'::set-output name=best_config::"{config_yml}"')
```

Comment:
that should be enough, right? You shouldn't even need that `entrypoint.sh` then?

Reply:
right